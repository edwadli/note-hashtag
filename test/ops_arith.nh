PrintEndline (StringOfInt (1 + 1))
PrintEndline (StringOfInt (4 - 2))
PrintEndline (StringOfInt (6 * 7))
PrintEndline (StringOfInt (5 / 2))
PrintEndline (StringOfInt (5 % 2))

PrintEndline (StringOfFloat (1.0 + 1.0))
PrintEndline (StringOfFloat (4.0 - 2.0))
PrintEndline (StringOfFloat (6.0 * 7.0))
PrintEndline (StringOfFloat (5.0 / 2.0))
