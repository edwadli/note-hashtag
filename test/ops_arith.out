2
2
42
2
1
2.000000
2.000000
42.000000
2.500000
